crates/sargon/src/system/sargon_os/transactions/sargon_os_transaction_analysis.rs

```diff
@@ -537,20 +537,20 @@ mod transaction_preview_analysis_tests {
             TransactionPreviewResponse {
                 encoded_receipt: "".to_string(),
                 radix_engine_toolkit_receipt: Some(ScryptoSerializableToolkitTransactionReceipt::CommitSuccess {
-                    state_updates_summary: native_radix_engine_toolkit::receipt::StateUpdatesSummary {
+                    state_updates_summary: radix_engine_toolkit_common::receipt::StateUpdatesSummary {
                         new_entities: IndexSet::new(),
                         metadata_updates: IndexMap::new(),
                         non_fungible_data_updates: IndexMap::new(),
                         newly_minted_non_fungibles: IndexSet::new(),
                     },
                     worktop_changes: IndexMap::new(),
-                    fee_summary: native_radix_engine_toolkit::receipt::FeeSummary {
+                    fee_summary: radix_engine_toolkit_common::receipt::FeeSummary {
                         execution_fees_in_xrd: ret_zero,
                         finalization_fees_in_xrd: ret_zero,
                         storage_fees_in_xrd: ret_zero,
                         royalty_fees_in_xrd: ret_zero,
                     },
-                    locked_fees: native_radix_engine_toolkit::receipt::LockedFees {
+                    locked_fees: radix_engine_toolkit_common::receipt::LockedFees {
```

Comment:
consider adding to `lib.rs` a `radix_engine_toolkit_common::receipt::LockedFees as RETLockedFees` and use just `RETLockedFees` here.

Reply:
or if it is **only** used in this file, no need to add it to `lib.rs`, but at the top of this file instead.

Reply:
Only used in this file. Added at the top.